CONTRIBUTING.md

```diff
@@ -17,26 +84,222 @@ augroup gno_autocmd
+| breaking change | PR introduces backwards incompatible functionality              |
+| bug fix         | PR introduces a fix for a known bug                             |
+| dependencies    | PR introduces a package version bump                            |
+| documentation   | PR introduces an improvement or addition to the docs            |
+| don’t merge     | PR contains unstable functionality that shouldn't be merged yet |
+| feature         | PR adds new functionality to Gno                                |
+| hotfix          | PR applies a hotfix that should be merged ASAP                  |
+
+#### Labels for Pull Requests
+
+| Label Name       | Description                                       |
+|------------------|---------------------------------------------------|
+| bug              | Issue confirms a bug is present in Gnoland        |
+| duplicate        | Issue already exists, or has been posted before   |
+| good first issue | Issue is a great introduction for newcomers       |
+| help wanted      | Issue requires extra attention from the community |
+| info needed      | Issue is lacking information needed for resolving |
+| investigating    | Issue is still being investigated by the team     |
+| question         | Issue starts a discussion or raises a question    |
+
+### Notable Contributions
 
 Notable contributions of fixes/features/refactors:
 
 * [https://github.com/gnolang/gno/pull/208](#208) - @anarcher, gnodev test with testing.T
 * [https://github.com/gnolang/gno/pull/167](#167) - @loicttn, website: Add syntax highlighting + security practices
 * [https://github.com/gnolang/gno/pull/136](#136) - @moul, foo20, a grc20 example smart contract
-* [https://github.com/gnolang/gno/pull/126](#126) - @moul, feat: use the new Precompile in gnodev and in the addpkg/execution flow (2/2)
+* [https://github.com/gnolang/gno/pull/126](#126) - @moul, feat: use the new Precompile in gnodev and in the
+  addpkg/execution flow (2/2)
 * [https://github.com/gnolang/gno/pull/119](#119) - @moul, add a Gno2Go precompiler (1/2)
 * [https://github.com/gnolang/gno/pull/112](#112) - @moul, feat: add 'gnokey maketx --broadcast' option
-* [https://github.com/gnolang/gno/pull/110](#110), [https://github.com/gnolang/gno/pull/109](#109), [https://github.com/gnolang/gno/pull/108](#108), [https://github.com/gnolang/gno/pull/106](#106), [https://github.com/gnolang/gno/pull/103](#103), [https://github.com/gnolang/gno/pull/102](#102), [https://github.com/gnolang/gno/pull/101](#101) - @moul, various chores.
+* [https://github.com/gnolang/gno/pull/110](#110), [https://github.com/gnolang/gno/pull/109](#109), [https://github.com/gnolang/gno/pull/108](#108), [https://github.com/gnolang/gno/pull/106](#106), [https://github.com/gnolang/gno/pull/103](#103), [https://github.com/gnolang/gno/pull/102](#102), [https://github.com/gnolang/gno/pull/101](#101) -
+  @moul, various chores.
```

Comment:
I don't think the change is better.

We'll remove this section soon, thanks to #407.

Reply:
Reverted the formatting for this section:

[929b437](https://github.com/gnolang/gno/pull/469/commits/929b437c93fde1a6c76bb2fd5f2ed08914950afa)